docs/quickstart/install.md

```diff
@@ -40,16 +40,16 @@ Apache Zeppelin officially supports and is tested on the following environments:
   </tr>
   <tr>
     <td>OS</td>
-    <td>Mac OSX <br /> Ubuntu 16.X</td>
+    <td>Mac OSX <br/>Ubuntu 16.X <br/> Ubuntu 18.X <br/> Ubuntu 20.X</td>
```

Comment:
I think we should remove Ubuntu 16.X because the end of standard support is in April 2021.
The 'X' should be changed to '04'. For more information, see https://wiki.ubuntu.com/Releases.

Reply:
Thanks @Reamer , I have updated it